indico/modules/categories/controllers/display.py

```diff
@@ -453,8 +457,7 @@ def _process_args(self):
             self.end_dt = self.start_dt + relativedelta(months=1)
 
     def _process(self):
-        info = get_category_timetable([self.category.id], self.start_dt, self.end_dt, detail_level=self.detail,
-                                      tz=self.category.display_tzinfo, from_categ=self.category, grouped=False)
+        info = self._get_data()
```

Comment:
Now that I'm looking at this again, `_get_timetable` would probably be a better name ;)

Reply:
OK, i'll change it to this.